extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/user/CASAuthenticatedUser.java

```diff
@@ -79,9 +85,10 @@ public void init(String username, Credentials credentials) {
      *     as tokens when connections are established with this user.
      */
     public void init(String username, Credentials credentials,
-            Map<String, String> tokens) {
+            Map<String, String> tokens, Set<String> effectiveGroups) {
```

Comment:
If adding a new parameter, documentation needs to be added for that parameter (see the documentation for other parameters of this function for examples of this).

Reply:
ok, will fix.